packages/account/src/Components/icon-message-list/__tests__/icon-message-list.spec.tsx

```diff
@@ -67,7 +67,9 @@ describe('<IconMessageList/>', () => {
         renderComponent({ props: new_props });
         const upload_btn = screen.queryByRole('button', { name: /verify again/i });
         expect(upload_btn).toBeInTheDocument();
-        userEvent.click(upload_btn);
+        if (upload_btn) {
```

Comment:
may I clarify what is the issue here? why do we need condition?

Reply:
This was to overcome type error. The element's reference is fetched using `queryByRole`
There were quite a number of TS related errors that were not being rendered before. The config changer has ensured type safety
<img width="964" alt="Screenshot 2024-05-29 at 7 47 33 AM" src="https://github.com/binary-com/deriv-app/assets/98398322/93da4db0-070c-448e-adba-c64810cdc55d">